### ISSUE TITLE
Fix errors in put_characteristic.py after successfully putting values

### ISSUE THREAD

**Issue by jlusiardi** [jlusiardi/homekit_python#66]
E.g. `PYTHONPATH=. python3 homekit/put_characteristic.py   -f ble.json -a A60 -c 20.15 on` causes the following exception even on success:

```
Traceback (most recent call last):
  File "homekit/put_characteristic.py", line 71, in <module>
    for key, value in results.items():
AttributeError: 'NoneType' object has no attribute 'items'
```
This should not happen and return something similar to the IP based accessories

**Comment by Jc2k** [comment]
This should be fixed now #69 is merged

**Comment by jlusiardi** [comment]
thank you
